localizations.yaml

```diff
@@ -55,6 +55,11 @@ ANDROID:
       COPY:
         en: you
         nl: Zelf
+  ManageAccount:
+    CreatedAt:
+      COPY:
+        en: "EEEE, dd MMMM yyyy 'at' HH:MM"
```

Comment:
This is a java-specific date format. Can't you format this in the Android side, and only get the "at" part from the localization (it should already be there IIRC)